setup.py

```diff
@@ -69,15 +69,11 @@
 if project_name == TFA_RELEASE:
     # TODO: remove if-else condition when tf supports package consolidation.
     if platform.system() == 'Linux':
-        REQUIRED_PACKAGES.append('tensorflow-gpu == 2.0.0-rc0')
+        REQUIRED_PACKAGES.append('tensorflow-gpu == 2.1.0')
```

Comment:
Make this `2.0.0`? Without knowing the schedule for `2.1.0` it's possible we have another release on 2.0.

Reply:
Sure, done :-)